server/lib/game/chunkGenerator.go

```diff
@@ -166,7 +166,7 @@ func EachChunkNearby(wc coords.World, cb func(cc coords.Chunk, priority int)) {
 	}
 
 	cc := wc.Chunk()
-	eachWithin(cc, 2, 1, 2, func(newCC coords.Chunk, dist int) {
+	eachWithin(cc, 10, 3, 10, func(newCC coords.Chunk, dist int) {
```

Comment:
I'm reverting this now. This was changed just so I could quickly check far away for issues.